tcf_website/templates/course/course.html

```diff
@@ -17,6 +17,7 @@
             <h1 class="mr-md-3">{{ course.code }}</h1>
             <h2>{{ course.title }}</h2>
         </div>
+        <button type="button" ><a href={{course.eval_link}}>Student Evaluations (Requires NetBadge login)</a></button>
```

Comment:
Would recommend you have the link open up in a new tab!

Reply:
Done!